Dockerfile

```diff
@@ -37,6 +41,8 @@ RUN apk --update add build-base ca-certificates ruby ruby-dev && \
     apk del build-base ca-certificates ruby ruby-dev && \
     rm /var/cache/apk/*
 
+COPY . /usr/src/app
```

Comment:
The second argument here can also change to `./` since the `WORKDIR` is still `/usr/src/app`